media/js/app/assetmgr/assetpanel.js

```diff
@@ -22,31 +22,32 @@ var AssetPanelHandler = function(el, parent, panel, space_owner) {
     var self = this;
 
     self.el = el;
```

Comment:
Can self.el be removed now in favor of self.$el? (also in collection.js, discussionpanel.js, projectpanel.js)